lib/logstash/codecs/compress_spooler.rb

```diff
@@ -6,12 +6,18 @@ class LogStash::Codecs::CompressSpooler < LogStash::Codecs::Base
   config :spool_size, :validate => :number, :default => 50
   config :compress_level, :validate => :number, :default => 6
 
+  # The amount of time in seconds since last flush before a flush is forced,
+  # on the next event.
+  # Values smaller than 0 disables time based flushing.
+  config :min_flush_time, :validate => :number, :default => 0
```

Comment:
No, the default value of 0 does not change behavior. Setting min_flush_time to 0 disables time based flushing as the second part of the condition (after ||) on line 34 will never be true, because `@min_flush_time > 0 == false`. This means, the buffer is flushed, `if @buffer.length > @spool_size` is true. This is the old behavior.